modoboa_amavis/lib.py

```diff
@@ -395,3 +397,80 @@ def setup_manual_learning_for_mbox(mbox):
                 create_user_and_use_policy(alias, policy)
             result = True
     return result
+
+
+def split_address(address):
+    assert isinstance(address, six.text_type),\
+        "address should be of type %s" % six.text_type.__name__
+    if "@" not in address:
+        local_part = address
+        domain = ""
+    else:
+        local_part, domain = address.split("@", 1)
+    return (local_part, domain)
```

Comment:
Duplicate of https://github.com/modoboa/modoboa/blob/master/modoboa/lib/email_utils.py#L265 ?

Reply:
fixed, see modoboa/modoboa#1360